tests/integration_tests/test_applications_from_config.py

```diff
@@ -161,11 +161,21 @@ def validate_application_output(config):
             )
         if "OUTPUT_FILE" in integration_test:
             # In this case the output file is the file generated by the application
-            assert (
-                Path(config["CONFIGURATION"]["DATA_DIRECTORY"])
-                .joinpath(integration_test["OUTPUT_FILE"])
-                .exists()
-            )
+            # First check if the output is in the data directory (simtel_array related),
+            # Then check if the file is in the output directory (remaining tools).
+            print(Path(config["CONFIGURATION"]["OUTPUT_PATH"]))
```

Comment:
This is something we need to solve - the writing into output paths should be clarified. I remember you complained about it in Dec - did you open an issue?

Reply:
Ok, I just opened an issue for that https://github.com/gammasim/simtools/issues/760

The case here is a bit different. When running CORSIKA and simtel_array, the outputs are saved into the data_directory (corsika-data and simtel-data directories), while the remaining applications save their outputs into output_path. Perhaps we should reconsider saving CORSIKA and simtel_array results into the output directory or moving the output files to the output directory in the application code.

Reply:
We had in Dec a discussion on this (not sure if you where around) and we decided that we want to keep the separation between the output of simtools and those of corsika/sim_telarray. I don't like that it is called 'data' (as it is simulations), but this is to stay consistent with many of the historically grown scripts.

Reply:
This discussion is older than December as I raised it a couple of times before. Ok, then we focus on the use of `use_plain_output_path` in the new issue and deal with the output directory only

Reply:
One question remains though: what should we do with the output of simtel_array that is also the output of an application?

Reply:
Do we want to move this discussion somewhere else - I think it is clear to us now that we need to revisit how we write any data/output to disk.

Reply:
Ok, moved it to #760